eZ/Publish/Core/FieldType/Tests/CountryTest.php

```diff
@@ -568,8 +568,8 @@ public function provideValidDataForValidate()
      *              ),
      *          )
      *      ),
-     *      array(
-     *          new ValidationError(
+     *          array(
+     *              new ValidationError(
```

Comment:
This doesn't add up, it's an example anyway and alignment looked ok (more files affected)